samples/kotlin-ios-app/gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.jvmargs=-Xmx12g
 org.gradle.parallel=true
 org.gradle.caching=true
-org.gradle.configuration-cache=true
+org.gradle.configuration-cache=false
```

Comment:
we should enable it

Reply:
gradle.configuration-cache broken for watchos tasks in kotlin 1.9.10, but after upgrade kotlin version probles was fixed, enabled now